interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -710,6 +710,19 @@ public void gracefulShutdown() throws Exception {
     verifyNoMoreInteractions(responseObserver);
   }
 
+  /** Sends an rpc to an unimplemented method on the server. */
```

Comment:
Why was this moved?

If we are going to move it, I would hope that we would keep these in the same order as in interop-test-descriptions.md.